Documentation/project-docs/developer-guide.md

```diff
@@ -246,5 +246,16 @@ If coverage succeeds, the code coverage report will be generated automatically a
 
 Code coverage reports from the continuous integration system are available from the links on the front page of the corefx repo.
 
+### Building tests with .NET Native 
+
+.NET Native is a technology that allows compiling IL applications down into a native executable and minimal set of native DLLs, containing all needed functionality from the .NET Framework in native format.  For CoreFX tests, .NET Native support in CoreFX is relatively early, but supported.  
+
+```cmd
+// To run a single project with the .NET Native toolchain, set the appropriate build flags:
+cd src\Microsoft.CSharp\tests
+msbuild /t:BuildAndTest /p:TestTFM=netcore50aot  /p:TestNugetRuntimeId=win10-x64-aot /p:TestNugetRuntimeId=win10-x64-aot /p:FilterToTestTFM=netcore50 /p:UseDotNetNativeToolchain=true
```

Comment:
Why are you passing TestNugetRuntimeId twice? Actually why do we need to pass it at all? I also suspect you shouldn't need to pass FilterToTestTFM at all either. In fact it looks like you are in conflict between netcore50 and netcore50aot. What happens if you don't pass UseDotNetNativeToolchain? Shouldn't that be implied by the aot in the TestTFM?